web/src/pages/Plugin/locales/zh-CN.ts

```diff
@@ -16,5 +16,5 @@
  */
 export default {
   'page.plugin.drawer.popconfirm.title.delete': '确定删除该插件吗？',
-  'page.plugin.list': '插件列表'
+  'page.plugin.list': '插件列表：已启用插件的列表'
```

Comment:
Hi, as for the subtitle, how about using the `extra` prop like here[1]?

[1] https://procomponents.ant.design/en-US/components/page-container

Reply:
okay i have closed this and will try to fix this in next PR

Reply:
![screencapture-172-20-10-3-8000-plugin-list-2021-03-26-08_19_00](https://user-images.githubusercontent.com/40708551/112570678-86f6b900-8e0c-11eb-91e5-1510636a5c75.png)

is it cool now?